tests/e2e/vmc_upgrade_tkg.go

```diff
@@ -182,7 +183,13 @@ var _ = ginkgo.Describe("Upgrade TKG", func() {
 			wcpToken = getWCPSessionId(vmcWcpHost, e2eVSphere.Config.Global.VmcCloudUser,
 				e2eVSphere.Config.Global.VmcCloudPassword)
 		}
-		tkg_images := getVMImages(vmcWcpHost, wcpToken)
+		vmImage := getVMImages(vmcWcpHost, wcpToken)
+		size := len(vmImage.Items)
+
+		for i := 0; i < size; i++ {
+			framework.Logf(vmImage.Items[i].Spec.ProductInfo.Version)
+			tkg_images = append(tkg_images, vmImage.Items[i].Spec.ProductInfo.Version)
+		}
 
 		for _, image := range tkg_images {
```

Comment:
```golang
for _, image := range vmImage.Items {
   if image == image.Spec.ProductInfo.Version {
```
you can reduce one for loop this way, than was my intention when I asked you to return VMimages object

Reply:
done